### DISCUSSION TITLE
importing graphiql as local package

### DISCUSSION THREAD

**Discussion by uchia-itachi** [graphql/graphiql#3477] [Q&A / Support]
HI all,

i want to work on  doc explorer part  of the graphiql package. but i'm facing issue   importing the package from my local path. 
can anyone please help me how can i import the local graphiql project  as a dependency in my project

**Comment by acao** [comment]
`@graphiql/react` is where this currently lives!

**Reply by uchia-itachi** [reply]
i have clone the repo  and i want to use it from local path i tried  npm i ../mylocalpath/graphiql but it's giving me error

**Reply by acao** [reply]
you should import from @graphiql/react
